### PR TITLE
Update Servo Example

### PR DESCRIPTION
**Pull request by Hannes7eicher**
## What This PR Changes
This PR updates the schmatics for the Servo example.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jacobhylen** [comment]
@Hannes7eicher We can close this, no?

**Comment by Hannes7eicher** [comment]
@jacobhylen I added some more information to the power section regarding capacitors. Let me know what you think.

File: content/learn/04.electronics/05.servo-motors/servo-motors.md [hunk 1]
```diff
@@ -17,26 +17,57 @@ You can also visit the [Servo GitHub repository](https://github.com/arduino-libr
...
+- power supply
+
+## Powering Servo Motors
+
+Servo motors have different power requirements depending on their size and the workload they are experiencing. A common servo motor such as the [Feetech Mini Servo Motor](https://store.arduino.cc/products/feetech-mini-servo-motor-120-degrees-9g) requires between 4.8 - 6 V at 5 – 6 mA when idle. But as soon as the motor experiences heavier loads such as added weight or an object blocking its movement the current consumption can reach up to 800 mA.
+
+This amount of current is significantly above the max ratings for any Arduino board. To avoid damaging our board we need to power the servo motor through an external power supply. Choosing the correct power supply depends on the servo motor you are using, so always check the specifications. Pay especially close attention to the:
+
+- **operating voltage range**
+- **idle current** (in mA) - consumption when **not** moving
```
Comment (jacobhylen) [inline content/learn/04.electronics/05.servo-motors/servo-motors.md:30]:
```suggestion
- **idle current** - consumption when **not** moving
```

File: content/learn/04.electronics/05.servo-motors/servo-motors.md [hunk 2]
```diff
@@ -17,26 +17,57 @@ You can also visit the [Servo GitHub repository](https://github.com/arduino-libr
...
+- **running current** (in mA) - consumption when moving freely
```
Comment (jacobhylen) [inline content/learn/04.electronics/05.servo-motors/servo-motors.md:31]:
```suggestion
- **running current** - consumption when moving freely
```

File: content/learn/04.electronics/05.servo-motors/servo-motors.md [hunk 3]
```diff
@@ -17,26 +17,57 @@ You can also visit the [Servo GitHub repository](https://github.com/arduino-libr
...
+- **stall current** (in mA) - consumption under max load or when blocked
```
Comment (jacobhylen) [inline content/learn/04.electronics/05.servo-motors/servo-motors.md:32]:
```suggestion
- **stall current** - consumption under max load or when blocked
```

File: content/learn/04.electronics/05.servo-motors/servo-motors.md [hunk 4]
```diff
@@ -17,26 +17,57 @@ You can also visit the [Servo GitHub repository](https://github.com/arduino-libr
...
+
+**_Note that USB wall chargers are limited to 500 mA (USB 2.0) or 900 mA (USB 3.0)._**
+
+If your project needs to move around freely without being attached to a power outlet you can also choose batteries to power the servo. If you need 5 V exactly you can use two 18650 Li-Ion batteries together with a step-down converter.
+
+A step-down converter is needed because 18650 Li-Ion batteries will give you around 7.4 V. The max current depends on the specific battery but most of them are designed to output above 1A which is enough to power our small servo.
+
+**_If you are using bigger or more servos make sure to check your power requirements accordingly._**
+
+**Capacitors** are recommended for powering servo motors. While not strictly necessary, they help stabilize the power supply, minimize voltage drops, and reduce electrical noise. The specific capacitor values may vary based on the servo motor's requirements, but including them is good practice for better performance and reliability.
```
Comment (jacobhylen) [inline content/learn/04.electronics/05.servo-motors/servo-motors.md:44]:
```suggestion
**Capacitors** are recommended for powering servo motors. They help stabilize the power supply, minimize voltage drops, and reduce electrical noise. The specific capacitor values may vary based on the servo motor's requirements, but including them is good practice for better performance and reliability.
```

File: content/learn/04.electronics/05.servo-motors/servo-motors.md [hunk 5]
```diff
@@ -17,26 +17,57 @@ You can also visit the [Servo GitHub repository](https://github.com/arduino-libr
...
+
+When using a [Feetech Mini Servo Motor](https://store.arduino.cc/products/feetech-mini-servo-motor-120-degrees-9g) we recommend using a **100 µF** capacitor.
+
+![Capacitor](./assets/capacitor.png)
+
+**_When using capacitors, make sure to connect them correctly by checking for markings such as a white stripe, a '+' symbol, or a longer lead. Match these indicators with the corresponding signs on the capacitor and in your circuit (pay attention to the + and - signs), and be careful not to exceed the voltage limits. This precaution helps prevent issues like leaks or damage that could harm your circuit._**
```
Comment (jacobhylen) [inline content/learn/04.electronics/05.servo-motors/servo-motors.md:50]:
```suggestion
**_Because some capacitors are polarised (meaning that they have a direction), you may need to be careful with how you connect them to your circuit. Make sure to connect them correctly by checking for markings such as a white stripe, a '+' symbol, or a longer lead. If your capacitor has these, match the indicators of the capacitor with your circuit (pay attention to the + and - signs), and be careful not to exceed the voltage limits. This precaution helps prevent issues like leaks or damage that could harm your circuit._**
```

**Review by jacobhylen — COMMENTED** [review]
Nice. Big improvement to the article. 
Left nitpicky comments (sorry) :)

**Pull request merged**